src/headlessMoneyHash.ts

```diff
@@ -642,14 +684,15 @@ export default class MoneyHashHeadless<TType extends IntentType> {
   async renderUrl(
     url: string,
     renderStrategy: "IFRAME" | "POPUP_IFRAME" | "REDIRECT",
+    options?: RenderOptions,
```

Comment:
Can we make it generic, to add options only for the redirect and never for others?
or function overloading will be better also